bridges/otellogrus/hook_test.go

```diff
@@ -421,7 +426,7 @@ func assertAttributes(t *testing.T, want, got log.Record) {
 	}
 }
 
-func assertRecords(t *testing.T, want, got []log.Record) {
+func assertRecords(t *testing.T, want, got []logtest.EmittedRecord) {
```

Comment:
I think we can remove this function and use https://pkg.go.dev/go.opentelemetry.io/otel/log/logtest#AssertRecordEqual instead.